test/python/test_api_ibmq.py

```diff
@@ -59,6 +59,7 @@ def setUp(self):
         super().setUp()
         qiskit.register(QE_TOKEN, QE_URL, hub=QE_HUB, group=QE_GROUP,
                         project=QE_PROJECT)
+        # TODO: FIXME: Change this backend name when changed in IBM-Q
         self.backend = 'ibmqx_qasm_simulator'
```

Comment:
should the x be removed here

Reply:
This tests are for testing hub/group/project parameters... which only applies to commercial (IBM-Q), and commercial environment is not affected by the renaming of the simulators yet.